Makefile.dapper

```diff
@@ -3,11 +3,13 @@
 
 PROJECT ?= $(notdir $(CURDIR))
 BASE_DAPPER := Dockerfile.dapper
+USING = $(subst $(,), ,$(using))
```

Comment:
Interesting approach, though I wonder if you also pass `USING` when calling from outside dapper if this will hold or break :)

Reply:
Yeah no that's not what I meant but OK :)

I was talking about what would happen if someone would theoretically try running `make e2e USING="bla bla"` since it would be both an env var and a make flag but it was just a random thought about the weirdness of Make ;)

Reply:
Ah I gotcha, yeah I'm not sure about that, we set and export USING within the Makefile so hopefully anything user provided would be overridden!